src/dao/DAO.js

```diff
@@ -188,6 +189,45 @@ class DAO<TEntity: { id: EntityID }, TEntityMutator> extends BaseDAO<
     );
   }
 
+  // todo fix flow inconsistensy TResult with entityLoaders - TEntity
```

Comment:
may be for avoiding type collisions just make another map for custom queries cache?